apps/server/src/modules/deletion/repo/scope/deletion-request-scope.spec.ts

```diff
@@ -32,40 +32,59 @@ describe(DeletionRequestScope.name, () => {
 		});
 	});
 
-	describe('byStatus', () => {
+	describe('byStatusAndDate', () => {
 		const setup = () => {
-			const fifteenMinutesAgo = new Date(Date.now() - 15 * 60 * 1000); // 15 minutes ago
-			const expectedQuery = {
-				$or: [
-					{ status: StatusModel.FAILED },
-					{
-						$and: [
-							{ status: [StatusModel.REGISTERED, StatusModel.PENDING] },
-							{ updatedAt: { $lt: fifteenMinutesAgo } },
-						],
-					},
-				],
+			const newerThan = new Date(Date.now() - 15 * 60 * 1000); // 15 minutes ago
```

Comment:
setting variable instead of comment? (same for below)

Reply:
done